test/EVMHost.cpp

```diff
@@ -418,15 +440,19 @@ evmc::result EVMHost::precompileECRecover(evmc_message const& _message) noexcept
 				"debaaa0cddb321b2dcaaf846d39605de7b97e77ba6106587855b9106cb104215"
 				"61a22d94fa8b8a687ff9c911c844d1c016d1a685a9166858f9c7c1bc85128aca"
 			),
-			fromHex("0000000000000000000000008743523d96a1b2cbe0c6909653a56da18ed484af")
+			{
+				fromHex("0000000000000000000000008743523d96a1b2cbe0c6909653a56da18ed484af"),
+				gas_cost
+			}
 		}
 	};
 	evmc::result result = precompileGeneric(_message, inputOutput);
 	// ECRecover will return success with empty response in case of failure
-	if (result.status_code != EVMC_SUCCESS)
+	if (result.status_code != EVMC_SUCCESS && result.status_code != EVMC_OUT_OF_GAS)
```

Comment:
Looks like `EVMC_OUT_OF_GAS` case is a bug fix? I.e. before it would be replaced by success, which was incorrect?

I'd put it in a separate commit if that's the case here.